backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -30,12 +30,70 @@ public class OrgMutations
             Members =
             [
                 new OrgMember() { Role = OrgRole.Admin, UserId = userId }
-            ]
+            ],
+            Projects = []
         });
         await dbContext.SaveChangesAsync();
         return dbContext.Orgs.Where(o => o.Id == orgId);
     }
 
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> AcquireProject(
+        LexBoxDbContext dbContext,
+        IPermissionService permissionService,
+        Guid orgId,
+        Guid projectId)
+    {
+        var org = await dbContext.Orgs.FindAsync(orgId);
+        NotFoundException.ThrowIfNull(org);
+        permissionService.AssertCanEditOrg(org);
+        var project = await dbContext.Projects.FindAsync(projectId);
+        NotFoundException.ThrowIfNull(project);
+        permissionService.AssertCanManageProject(projectId);
+
+        if (await dbContext.OrgProjects.AnyAsync(op => op.OrgId == orgId && op.ProjectId == projectId))
+        {
+            // No error since we're already in desired state; just return early
+            return dbContext.Orgs.Where(o => o.Id == orgId);
+        }
+        await dbContext.OrgProjects.AddAsync(new OrgProjects { OrgId = orgId, ProjectId = projectId });
+        await dbContext.SaveChangesAsync();
+        return dbContext.Orgs.Where(o => o.Id == orgId);
+    }
+
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> ReleaseProject(
```

Comment:
yes it's valid for a project to not be in an org, I don't see is changing that long term.

Reply:
Another way to model this is to require all projects to be in an org and then have a default "global org" with appropriate administrators.  This might be nice to have some admins with broader powers who are not devs / super admins.

Reply:
I agree that projects with no org owner are fine. A "global org" might be tricky, because it would need special handling. With all *other* orgs, if a second org wants to own the project, we'd allow that without removing the project from the first org. But if the first owner was the global org, then we'd want to automatically remove the project from the global org whenever another org acquires it. And when any project was released from an org, we'd have to check if that was the last owner and reassign the ownership if it was. I don't see the benefits as being worth the complexity costs of having a "global org" to pseudo-own all unowned projects.

If we want to create a tier of admins with broad but not omnipotent powers, we can probably do that more easily by creating another user category besides "User" and "Site admin".

Reply:
A great name for that role would be [libero](https://en.wikipedia.org/wiki/Volleyball#Libero) 😄:
> the libero must wear a contrasting jersey color from their teammates and cannot block or attack the ball when it is entirely above net height. When the ball is not in play, the libero can replace any back-row player, without prior notice to the officials.

...but let's not call them that 🤣.

Reply:
Planning to close this conversation as resolved, and not implement a "global" org, unless someone chimes in to say otherwise in the next 24 hours or so.

Reply:
No I don't want to have a global org that would just result in a lot of complicated code where we need to treat that org as special.